ch5.md

```diff
@@ -227,11 +231,10 @@ var f = function(x){ return x === 4; };
 var isFourLetterWord = compose(f, g);
 ```
 
-**A distinguished morphism called identity**
-Let's introduce a useful function called `id`. This function simply takes some input and spits it back at you. Take a look:
+**A distinguished morphism called identithpintroduce a useful function called `id`. This function simply takes some input and spits ihpke a look:
```

Comment:
Just noticed something odd here is all. Maybe some spaces got deleted or something?